service/src/main/java/org/kiwiproject/champagne/model/DeployableSystem.java

```diff
@@ -37,12 +37,26 @@ public class DeployableSystem {
     @With
     List<SystemUser> users;
 
+    /**
+     * Transitive property that is the resolved development environment name.
```

Comment:
What do you mean by "Transitive property"?

Reply:
Meaning it is not stored directly in the database. I augment the record with extra data that goes in those extra fields

Reply:
OK, so then what it should really say is _transient_ right?

Reply:
Yep, fixed and pushed